### PR TITLE
Created the backend, as well as connected the form

### LINKED ISSUE #7: Use the slack api to onboard members
Create a service that will use the slack api, and then call that service within a controller to send our user to slack's api to have them join our slack.

This guide will prove useful:
https://levels.io/slack-typeform-auto-invite-sign-ups/

### PR DESCRIPTION
**Pull request by torch2424**
closes #7 
may close the issue #8 
may close the issue #13 

Firstly, let me apologize for closing multiple issues in one branch, the other two issues need some more testing, and could be confirmed by anyone willing to serve up the site and test things out :)

I created a little node.js backend to ping some services for us (Such as ifttt for google sheets, an undocumented slack api endpoint to auto invite emails, and the github api to automate org invintes), that we can access from our frontend form, using a little AngularJS service. This was tested working well (Would be nice to do some additional testing however).

This also does some various frontend design improvements, and a lot of them were mentioned by @davidnuon  in #13 , and this fixes a lot of those. They mostly dealt with responsiveness, but any additional testing, and confirmation would be nice to confirm closing the issue.

Screenshot (Chrome Inspected Nexus 5x): 

screenshot-localhost 9000 2016-03-06 02-04-32

P.S this does contain some testing code, such as localhost backend urls and things, that i left in, simply for easier testing by anyone wanting to check this out. Please let me know if this a problem, and I can remove it

### CODE REVIEW

**Comment by davidnuon** [comment]
Still reviewing the code, I can see one change that can be done. Please turn the coffee cup into an image.

Other than that, looking good so far!

screen shot 2016-03-07 at 9 25 28 pm

File: backend/routes/index.js [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+
+//Read our keys.json file
+var fs = require("fs");
+
+//Parse our keys.json
+keys = JSON.parse(fs.readFileSync('keys.json', 'utf8'));
+
+/* GET home page. */
+router.get('/', function(req, res, next) {
+  res.render('index', { title: 'I am a Hamburger' });
```
Comment (davidnuon) [inline backend/routes/index.js:15]: lol

Comment (davidnuon) [inline backend/routes/index.js:15]: nom nom nom

**Comment by davidnuon** [comment]
Another kink in the animation in Chrome on OSX 
coffee_kink

File: backend/routes/index.js (hunk 1 above)
Comment (torch2424) [inline backend/routes/index.js:15]: What does that mean? Like I said I am a terrible backend guy, so any advice on what I did wrong would be cool! :)

**Comment by torch2424** [comment]
I made your changes, thank you for the help! @davidnuon

**Comment by torch2424** [comment]
And it is quite weird OSX chrome was behaving like this, I tested on my chrome in linux, and on android and had no problems...

File: backend/routes/index.js (hunk 1 above)
Comment (rogerhoward) [inline backend/routes/index.js:15]: I think he's just cracking up at the throwaway response to someone accessing /

Personally, I like returning HTTP 418 in this scenario :)

Comment (torch2424) [inline backend/routes/index.js:15]: OHHHHHHH I forgot I put that in there. Yeah julian and I have a joke, similar to googl'es teapot:

http://www.error418.net/

And we can definitely add a return of 418 haha

**Comment by davidnuon** [comment]
All good :)

**Pull request merged**